csharp/OpenStudio.targets

```diff
@@ -1,23 +1,9 @@
-<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
The purpose of OpenStudio.targets is to define msbuild build commands that will cause projects using the openstudio nuget package, to copy the required native dlls into the consuming project's build output folder. These changes are updates to support 32/64 bit versions of the native libs.